TickSync.py

```diff
@@ -88,13 +90,17 @@ def tick(self):
         passed_time = (now - self._sim_start_clock).total_seconds()
         self.sim_time =  self.sim_start_time + passed_time
         self.tick_count+=1
+        GlobalTick()
```

Comment:
this notion of GlobalTick. How is it related to the global traffic rate? SimTraffic is always running at higher frequency than every vehicle planner. Why aren't we checking all violations in SimTraffic for all vehicles instead of each vehicle checking itself?